justfile

```diff
@@ -159,10 +165,14 @@ dart_linter mode="default":
     just dart_linter_pana
 
 _dart_linter_single mode directory executable line_length:
-    cd {{directory}} && dart format \
+    dart format {{directory}} \
+      --line-length {{line_length}} \
+      --fix \
+
+    dart format {{directory}} \
       --line-length {{line_length}} \
-      {{ if mode == "fix" { "--fix" } else { "--output=none --set-exit-if-changed" } }} \
-      .
+      --output=none --set-exit-if-changed \
+
```

Comment:
it seems we should not unconditionally call the `--fix`? because we seem to be using it in CI. thus, if we always fix it, CI will not detect any format error

Reply:
>  because we seem to be using it in CI.  thus, if we always fix it, CI will not detect any format error

Frankly, the reason that I make it always `fix` is exactly because there is an issue detected by CI but not detected locally by `just precommit`, so why not just let it always fix it?  Afterall,  there is no need to care if we make or not a lint issue, as long as there is no syntax lint issue,